src/components/table/index.md

```diff
@@ -56,3 +56,13 @@ If the [width override classes](/styles/layout/#width-override-classes) do not m
 To learn more about this, read guidance on [extending and modifying components in production](/get-started/extending-and-modifying-components/).
 
 {{ example({ group: "components", item: "table", example: "column-widths-custom-classes", html: true, nunjucks: true, open: false, size: "m" }) }}
+
+## Tables with a lot of data
+
+If possible, you should aim to have less data in your tables. If you have a lot of data, try to organise it into multiple tables or multiple pages.
+
+If you cannot split your data, you can use the CSS class `govuk-table--small-text-until-tablet`. This class reduces the size of the text on small screens so that the large amount of data is easier to read on smaller screens.
+
+You should not use this class on tables unless your table has a lot of data because a smaller amount of data is easier to read if the text is larger.
```

Comment:
```suggestion
You should not use this class on tables unless your table has a lot of data, because a smaller amount of data is easier to read if the text is larger.
```